plugins/gradle/src/main/java/org/wildfly/swarm/plugin/gradle/PackageTask.java

```diff
@@ -273,7 +273,7 @@ private File getPropertiesFile() {
 
     @OutputFile
     private File getOutputFile() {
-        return BuildTool.getOutputFile(getBaseName(), getOutputDirectory());
+        return BuildTool.getOutputFile(getBaseName() + "-swarm.jar", getOutputDirectory());
```

Comment:
How does Gradle get the `-swarm.jar` ending with this change?

The logic below is in the Maven plugin only

Reply:
I don't know if i misunderstand you here, but you're looking at this change. BuildTool.getOutputFile is called from PackageTask.getOutputFile in the wildfly-swarm-gradle-plugin. Since I moved the name generation to one place for maven builds (PackageMojo.java) and thus removed the +"-swarm.jar" from the BuildTool.getOutputFile I also had to add the +"-swarm.jar" in the PackageTask.getOutputFile to keep the behaviour consistent with the previous implementation.

Reply:
Sorry, was getting confused between removals and additions!